server/lib/nl/common/constants.py

```diff
@@ -295,6 +295,14 @@
         "RCP 4.5 (intermediate), °C",
     "ProjectedMin_Until_2050_DifferenceRelativeToBaseDate1981To2010_Min_Temperature_RCP60":
         "RCP 6.0 (slightly pessimistic), °C",
+    "DiffRelativeToAvg_1980_2010_MaxTemp_Daily_Hist_5PctProb_Greater_Atleast1DayADecade_CMIP6_Ensemble_SSP245": 
+        "SSP 2, RCP 4.5 (intermediate), °C",
```

Comment:
should we call it "SSP 245"?

https://screenshot.googleplex.com/6RNemFwBwRgtdLf

Reply:
Done